ci/steps/brew-bump.sh

```diff
@@ -23,68 +20,6 @@ main() {
     exit 1
   fi
 
-  # Make sure the git clone step is successful
-  if ! directory_exists "$REPO"; then
-    echo "git clone failed. Cannot find $REPO directory."
-    ls -la
-    exit 1
-  fi
-
-  echo "Changing into $REPO directory"
-  pushd "$REPO" && pwd
-
-  echo "Adding $UPSTREAM_USERNAME_AND_REPO"
-  git remote add upstream "https://github.com/$UPSTREAM_USERNAME_AND_REPO.git"
-
-  # Make sure the git remote step is successful
-  if ! git config remote.upstream.url > /dev/null; then
-    echo "git remote add upstream failed."
-    echo "Could not find upstream in list of remotes."
-    git remote -v
-    exit 1
-  fi
-
-  # TODO@jsjoeio - can I somehow check that this succeeded?
-  echo "Fetching upstream $UPSTREAM_USERNAME_AND_REPO commits"
-  git fetch upstream master
-
-  # TODO@jsjoeio - can I somehow check that this succeeded?
-  echo "Merging in latest $UPSTREAM_USERNAME_AND_REPO changes branch master"
-  git merge upstream/master
-
-  # GIT_ASKPASS lets us use the password when pushing without revealing it in the process list
-  # See: https://serverfault.com/a/912788
-  PATH_TO_GIT_ASKPASS="$HOME/git-askpass.sh"
-  # Source: https://serverfault.com/a/912788
-  # shellcheck disable=SC2016,SC2028
-  echo 'echo $HOMEBREW_GITHUB_API_TOKEN' > "$PATH_TO_GIT_ASKPASS"
-
-  # Make sure the git-askpass.sh file creation is successful
-  if ! file_exists "$PATH_TO_GIT_ASKPASS"; then
-    echo "git-askpass.sh not found in $HOME."
-    ls -la "$HOME"
-    exit 1
-  fi
-
-  # Ensure it's executable since we just created it
-  chmod +x "$PATH_TO_GIT_ASKPASS"
-
-  # Make sure the git-askpass.sh file is executable
-  if ! is_executable "$PATH_TO_GIT_ASKPASS"; then
-    echo "$PATH_TO_GIT_ASKPASS is not executable."
-    ls -la "$PATH_TO_GIT_ASKPASS"
-    exit 1
-  fi
-
-  # NOTE: we need to make sure our fork is up-to-date
-  # otherwise, brew bump-formula-pr will use an
-  # outdated base
-  echo "Pushing changes to $GITHUB_USERNAME/$REPO fork on GitHub"
-  # Export the variables so git sees them
-  export HOMEBREW_GITHUB_API_TOKEN="$HOMEBREW_GITHUB_API_TOKEN"
-  export GIT_ASKPASS="$PATH_TO_GIT_ASKPASS"
-  git push "https://$GITHUB_USERNAME@github.com/$GITHUB_USERNAME/$REPO.git" --all
-
```

Comment:
Based on [this comment](https://github.com/Homebrew/discussions/discussions/1532#discussioncomment-782540), using that setup homebrew action means we shouldn't have to keep a fork up-to-date and in testing things, it worked.

And to confirm, you can see in the PR description that `brew bump-formula-pr` with `--dry-run` worked even though `cdrci`'s fork is behind

<img width="554" alt="image" src="https://user-images.githubusercontent.com/3806031/159807502-7be78cc4-5dfe-435b-87c2-3ef0bc19a2a2.png">

Reply:
Wow, great news!  I guess the issue we ran into was an edge case.